tests/functional/storageclass/test_create_storageclass_with_same_name.py

```
@@ -60,16 +60,17 @@ def create_storageclass(sc_name, expect_fail=False):
-        log.debug(sc_data)
+        logger.info(f"Storage class: {SC_OBJ.name} created successfully")
+        logger.debug(f"Storage class data: {sc_data}")
 
     except CommandFailed as ecf:
         assert "AlreadyExists" in str(ecf)
-        log.info(
-            f"Cannot create two StorageClasses with same name !"
-            f" Error message:  \n"
-            f"{ecf}"
+        logger.info(
+            f"Cannot create two StorageClasses with same name. " f"Error message: {ecf}"
```

Ruff (0.15.15)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)